pkg/utils/subscription.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"cmp"
+	"os"
+)
+
+var (
+	noobaaOperatorChannel = cmp.Or(os.Getenv("NOOBAA_OPERATOR_CHANNEL"), "latest")
```

Comment:
Does it make sense to have 'latest' as the default? I am not sure there is a channel can even be called latest

Reply:
there is no `latest` channel. it was a suggestion by you previously as a default value indicator for now until we can better address channeling for the dependency.